src/base/projects/Commit/CommitTeaser.svelte

```diff
@@ -94,11 +93,6 @@
     <CommitAuthorship {commit} noDelegate />
   </div>
   <div class="column-right">
-    {#if commit.context.committer}
```

Comment:
Why are we removing this entirely?

Reply:
most of the stuff being removed is because of errors from `npm run check`, and the fact that api is no longer returning them